src/Microsoft.Health.Dicom.Api/Features/ModelBinders/CsvModelBinder.cs

```diff
@@ -0,0 +1,53 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System;
+using System.Threading.Tasks;
+using EnsureThat;
+using Microsoft.AspNetCore.Mvc.ModelBinding;
+using Microsoft.Extensions.Primitives;
+
+namespace Microsoft.Health.Dicom.Api.Features.ModelBinders
+{
+    internal class CsvModelBinder : IModelBinder
+    {
+        protected virtual object DefaultValue { get; } = Array.Empty<string>();
+
+        public Task BindModelAsync(ModelBindingContext bindingContext)
+        {
+            EnsureArg.IsNotNull(bindingContext, nameof(bindingContext));
+            StringValues values = bindingContext.ValueProvider.GetValue(bindingContext.ModelName).Values;
+
+            if (values.Count == 0)
+            {
+                bindingContext.Result = ModelBindingResult.Success(DefaultValue);
+            }
+            else if (values.Count > 1)
```

Comment:
We previously support this scenario, this could cause regression issue. 
I don't think such request (`/studies?includeField=PatientName&IncludeField=PatientAge`) is very common, but it's better to still support.

Reply:
https://github.com/microsoft/dicom-server/blob/b8e8ec5202b1778415e05d9f4c3397e47a9d7695/src/Microsoft.Health.Dicom.Core/Features/Query/QueryParser.ParameterParser.cs#L18

Reply:
This does strangely seem allowed by the DICOMweb standard: http://dicom.nema.org/medical/dicom/current/output/html/part18.html#sect_8.3.4.3

> The request may contain one or more include parameters; however, if a parameter with the value of "all" is present, then other includefield parameters shall not be present.

I also now throw an error if "all" is specify with other fields